geode-docs/tools_modules/lucene_integration.html.md.erb

```diff
@@ -89,10 +89,10 @@ gfsh>create lucene index --name=indexName --region=/orders --field=customer,tags
 
 ``` pre
 // Create an index, specifying a custom analyzer for the second field
-// Note: "null" in the first analyzer position uses the default analyzer
+// Note: "DEFAULT" in the first analyzer position uses the default analyzer
 // for the first field
 gfsh>create lucene index --name=indexName --region=/orders
-  --field=customer,tags --analyzer=null,org.apache.lucene.analysis.bg.BulgarianAnalyzer
+  --field=customer,tags --analyzer=DEFAULT,org.apache.lucene.analysis.bg.BulgarianAnalyzer
 ```
 To use the entire value as a single field set the required `--field`
```

Comment:
The sentence that needs the comma has been completely rewritten for GEODE-3002. See PR https://github.com/apache/geode/pull/550 for the rewrite.  Not implementing, as it would only cause a conflict upon merging in GEODE-3002.